README.md

```diff
@@ -125,10 +125,42 @@ class HelloWorldSpec extends IntegrationSpec {
     }
 }
 ```
-The plugin sets default versions to test against, but these can be overridden using the `gradleTestUtils` extension.  e.g.
+
+# Setting Gradle test versions
+
+There are two ways to set which Gradle versions to test against:
+
+### The `gradle/gradle-test-versions.yml` file
+
+By default, all `@GradlePluginsTests` run against the versions specified in `gradle/gradle-test-versions.yml`. Using a yaml file allows excavator to easily roll out minor version bumps, and remove deprecated major versions.
```

Comment:
don't think it's necessary to talk about excavator here,

Reply:
actually, this is fine I guess.

I think we don't want to harp on about excavator explicitly, but largely just be like: the `major-versions` block is expected to be edited by robots where it will bump the versions to test against. The `extra-versions` block is expected to be edited by humans where they need to run a particular version to test specific behaviour